exonum-java-binding/app/src/main/java/com/exonum/binding/app/ServiceRuntimeBootstrap.java

```diff
@@ -87,7 +89,8 @@ public static ServiceRuntime createServiceRuntime(String serviceArtifactsDir,
           DEPENDENCY_REFERENCE_CLASSES);
       Injector frameworkInjector = Guice.createInjector(APP_STAGE, frameworkModule);
 
-      return frameworkInjector.getInstance(ServiceRuntime.class);
+      ServiceRuntime runtime = frameworkInjector.getInstance(ServiceRuntime.class);
+      return new ServiceRuntimeAdapter(runtime, ViewProxyFactory.getInstance());
```

Comment:
```suggestion
      return frameworkInjector.getInstance(ServiceRuntimeAdapter.class);
```

Reply:
suggested changes don't seem to work with multiline comments 🤷‍♂

Reply:
Yep, the feature is nice, but I hope the suggestions will be adjusted by github team. Thanks, will change this.